taxi/views.py

```diff
@@ -34,6 +36,23 @@
     paginate_by = 5
 
 
+class ManufacturerCreateView(CreateView):
+    model = Manufacturer
+    form_class = ManufacturerForm
+    success_url = reverse_lazy("manufacturer-list")
+
+
+class ManufacturerUpdateView(UpdateView):
+    model = Manufacturer
+    form_class = ManufacturerForm
+    success_url = reverse_lazy("manufacturer-list")
+
+
+class ManufacturerDeleteView(DeleteView):
```

Comment:
Consider adding `LoginRequiredMixin` to `ManufacturerDeleteView` to ensure that only authenticated users can delete a manufacturer.